### PR TITLE
Fix Linter

### PR DESCRIPTION
**Pull request by v2tmobile**
Eslint cleans!

### CODE REVIEW

File: server/controllers/bookmark.controller.js [hunk 1]
```diff
@@ -267,7 +267,7 @@ function unbookmark(req, res, next) {
       return Bluebird.all([newFavorite.save()]);
     })
     .then((favorite) => {
-      req.favorite = favorite[0]; // eslint-disable-line no-param-reassign
```
Comment (jasonify) [inline server/controllers/bookmark.controller.js]: The logic here seems to differ from the original?

**Comment by jasonify** [comment]
This looks good, thanks! I left you one comment so maybe you might know why that was changed?

File: server/controllers/auth.controller.js [hunk 2]
```diff
@@ -167,8 +164,8 @@ function loginWithEmail(req, res, next) {
  */
 
 function register(req, res, next) {
-  const username = req.body.username;
-  const password = req.body.password;
+  const { username } = req.body;
```
Comment (dfcook) [inline server/controllers/auth.controller.js]: This destructuring can be done in one line:

const { username, password } = req.body;

File: server/controllers/bookmark.controller.js (hunk 1 above)
Comment (v2tmobile) [reply server/controllers/bookmark.controller.js]: Yeah, you are right. I was mistake this line.

Comment (andrewmarklloyd) [reply server/controllers/bookmark.controller.js]: It doesn't look like we even use the `req.favorite`?

Comment (v2tmobile) [reply server/controllers/bookmark.controller.js]: It should be req.favorite = favorite[0];

**Pull request merged**